libr/core/canal.c

```diff
@@ -1831,6 +1831,12 @@ static int core_anal_graph_nodes(RCore *core, RAnalFunction *fcn, int opts, PJ *
 	char *pal_box4 = palColorFor ("graph.box4");
 	if (!fcn || !fcn->bbs) {
 		eprintf ("No fcn\n");
+		R_FREE (pal_jump);
```

Comment:
Just call free()

Reply:
R_FREE is free+ set Null. So if this gows before the return. Its extra work for the compiler to remove dead code